assets/scss/base/forms.scss

```diff
@@ -0,0 +1,47 @@
+.form-control {
+  background: rgba(0, 0, 0, 0);
+  border: 1px solid $aluminum;
+  border-radius: 5px;
+  display: block;
+  margin: auto;
+  padding: 2rem;
+  width: 35rem;
+}
+
+.button {
+  background: rgba(0, 0, 0, 0);
+  border: 1px solid $picton-blue;
+  border-radius: 5px;
+  color: $white;
+  margin-right: 19.5rem;
+  padding: 1rem;
+  width: 8.935rem;
+}
+
+.newsletter {
+  color: $white;
```

Comment:
Properties should be ordered border-radius, color, display, height, margin-left, margin-right, padding, width